### ISSUE TITLE
Broadcast Images are squished

### ISSUE THREAD

**Issue by HaonRekcef** [lichess-org/mobile#1308]
### What is the problem?

Broadcast images are twice as wide as they are tall, but we display them as squares in the watch tab.

### Steps to reproduce the bug

1. go to watch tab
2. observe images

### App version

0.13.12

### Device Info

-

**Comment by veloce** [comment]
That's because of the cache size optimisation. If we remove that optimisation they are not distorted anymore.

In any case I think they should be displayed square. I don't even care much if they are distorted since they are so small.

**Comment by HaonRekcef** [comment]
I think we disagree here. I find the squares visually unappealing, especially when there is text on the images. It looks distorted and doesn't do justice to the clean design of the rest of the app.

![grafik](https://github.com/user-attachments/assets/d7aafbf4-1211-4c17-bf20-a831a204bfe4)

**Comment by veloce** [comment]
Well yeah I get your point. On iOS they are much smaller that's why it doesn't bother me.

**Comment by veloce** [comment]
![Screenshot_1735127204](https://github.com/user-attachments/assets/05a95061-f584-45cd-b0d3-0e5ba487e516)

This is how it looks like with the image not distorted. I think it is still better to have a square image here, especially on iOS where the image is so small. There's not point in seeing the whole image imo, it is too small to read the details.

**Comment by HaonRekcef** [comment]
I don’t like this much either because the text gets cut off, to me this just doesn't look good. Could we slightly reduce the height and increase the width to make it twice the height instead?

**Comment by EmmetSchuler** [comment]
And maybe slightly round the edges, similar to in the broadcast list screen?

**Comment by veloce** [comment]
To me, the images are not really significant here, they are not meant to be displayed so small. The colors and shapes can be used to distinguish the broadcast though, that's why I put the thumbnails back.
If we try to keep the original aspect ratio they don't look good either, and are so small that we don't see anything.

**Comment by julien4215** [comment]
I agree with HaonRekcef that the images should have the original aspect ratio. Even if the images are small you can clearly see that they are distorted or that the they are cut off which doesn't look good.

I cannot test how it looks on iOS but if the images are really much smaller on iOS maybe we can change the aspect ratio only for Android.
